### DISCUSSION TITLE
Section - 10, PyTorch Paper Replicating, how to generate confusion matrix for the test set after using pretrained vit?

### DISCUSSION THREAD

**Discussion by itsdebjit** [mrdbourke/pytorch-deep-learning#824] [Q&A]
Section - 10, PyTorch Paper Replicating, After using pre-trained ViT how to get the confusion matrix of the same, and how to get other performance metrics like F-1 Score, Precision, Recall etc for the test set? How to see the test samples which are correctly predicted and which are not? I need the y_pred and y values. Please help me.****
